test/TestThreeHeapOrderingInvariant.t.sol

```diff
@@ -48,9 +49,24 @@ contract TestThreeHeapOrderingInvariant is Test {
         uint256 length = heap.length();
 
         for (uint256 i; i < length; ++i) {
-            assertTrue((i * 3 + 1 >= length || i * 3 + 1 >= heap.size() || heap.accountsValue(i) >= heap.accountsValue(i * 3 + 1))); // prettier-ignore
-            assertTrue((i * 3 + 2 >= length || i * 3 + 2 >= heap.size() || heap.accountsValue(i) >= heap.accountsValue(i * 3 + 2))); // prettier-ignore
-            assertTrue((i * 3 + 3 >= length || i * 3 + 3 >= heap.size() || heap.accountsValue(i) >= heap.accountsValue(i * 3 + 3))); // prettier-ignore
```

Comment:
Can you keep the ignore here pls ? The comment is not exactly the same for forge fmt, but I know that it exists

Reply:
´forgefmt: disable-line´

Reply:
why did you resolved this ?